source/common/http/header_map_impl.cc

```diff
@@ -73,7 +73,7 @@ HeaderString::HeaderString(HeaderString&& move_value) {
   }
   case Type::Inline: {
     buffer_.dynamic_ = inline_buffer_;
-    memcpy(inline_buffer_, move_value.inline_buffer_, string_length_ + 1);
+    memcpy(inline_buffer_, move_value.inline_buffer_, string_length_);
```

Comment:
I think you missed one below here on line 116 (I searched for '1' in this file).

Reply:
Thank you.